driver-db/ens160.toml

```diff
@@ -0,0 +1,34 @@
+manifest_version = "0.0.1"
+
+[meta]
+categories = ["Sensor::TVOC", "Sensor::eCO2"]
+datasheets = [
+    "https://www.sciosense.com/wp-content/uploads/2023/12/ENS160-Datasheet.pdf",
+]
+manufacturer = "ScioSense"
+names = ["ENS160"]
+part_numbers = ["ENS160"]
+
+kicad_symbol = ["ENS160-BGLM"]
+
+[[dev_boards]]
+connections = ["Qwiic", "StemmaQt"]
+link = "https://www.adafruit.com/product/5606"
+manufacturer = "Adafruit"
+name = "Adafruit ENS160 MOX Gas Sensor - Sciosense CCS811 Upgrade - STEMMA QT / Qwiic"
+
+[[dev_boards]]
+connections = ["Qwiic", "StemmaQt"]
+link = "https://www.adafruit.com/product/5606"
+manufacturer = "Sparkfun"
+name = "SparkFun Indoor Air Quality Sensor ENS160 for Arduino Qwiic"
+
+[[dev_boards]]
+connections = ["StemmaQt"]
+link = "https://www.adafruit.com/product/5606"
+manufacturer = "DFRobot"
```

Comment:
Could you add this manufacturer to the schema? https://github.com/tweedegolf/drive-rs/blob/main/backend/src/driver_db/boards.rs#L14